src/pkg/backup/details/details.go

```diff
@@ -236,6 +236,21 @@ func (dm DetailsModel) FilterMetaFiles() DetailsModel {
 	return d2
 }
 
+// FilterMetaFiles returns a copy of the Details with all of the
+// .meta files removed from the entries.
+func (dm DetailsModel) NonMetaFileSizes() int64 {
```

Comment:
The current name makes it sound like it returns each file size individually.
```suggestion
func (dm DetailsModel) SumNonMetaFileSizes() int64 {
```